roofit/hs3/CMakeLists.txt

```
@@ -0,0 +1,49 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT rootfit/histfactory package
+# @author Carsten Burgard, CERN
+############################################################################
+
+ROOT_STANDARD_LIBRARY_PACKAGE(RooFitHS3
+  HEADERS
+  JSONInterface.h
+  RooJSONFactoryWSTool.h
+  HistFactoryJSONTool.h
+  JSONParser.h
+  RYMLParser.h
+  SOURCES
+  src/RooJSONFactoryWSTool.cxx
+  src/HistFactoryJSONTool.cxx
+  src/JSONParser.cxx
+  src/RYMLParser.cxx
+  src/JSONFactories_RooFitCore.cxx
+  src/JSONFactories_HistFactory.cxx
```

```suggestion
    src/RooJSONFactoryWSTool.cxx
    src/HistFactoryJSONTool.cxx
    src/JSONParser.cxx
    src/RYMLParser.cxx
    src/JSONFactories_RooFitCore.cxx
    src/JSONFactories_HistFactory.cxx
```
Fix indentation (to make it consistent with the parameters below).